### PR TITLE
Remove empty Package.swift for swiftpm tests.

### PR DESCRIPTION
**Pull request by aciidb0mb3r**
swiftpm is removing support for empty manifest files. This PR updates
tests removing empty Package.swift

Needs to land together with https://github.com/apple/swift-package-manager/pull/432

### CODE REVIEW

**Comment by ddunbar** [comment]
@swift-ci please test

File: test-foundation-package/test-foundation-package.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # Make a sandbox dir.
 # RUN: rm -rf %t.dir
 # RUN: mkdir -p %t.dir/tool
-# RUN: touch %t.dir/tool/Package.swift
+# RUN: cp %S/Package.swift %t.dir/tool/Package.swift
 # RUN: cp %S/main.swift %t.dir/tool/main.swift
 # RUN: %{swift} build --chdir %t.dir/tool -v > %t.build-log
```
Comment (ddunbar) [inline test-foundation-package/test-foundation-package.py]: Might as well also make this file the `Package.swift` (same for below)

Comment (aciidb0mb3r) [inline test-foundation-package/test-foundation-package.py]: ok

Comment (aciidb0mb3r) [inline test-foundation-package/test-foundation-package.py]: done

**Comment by ddunbar** [comment]
@swift-ci please test

**Comment by aciidb0mb3r** [comment]
@ddunbar doesn't look like swift ci works for this repo?

**Comment by ddunbar** [comment]
Ah, right. I'll check manually.

**Comment by ddunbar** [comment]
LGTM

**Pull request merged**